packages/browser/test/index.test.ts

```diff
@@ -1,5 +1,4 @@
-import BugsnagBrowserStatic from '..'
-import { Breadcrumb, Session } from '../types/bugsnag'
+import BugsnagBrowserStatic, { Breadcrumb, Session } from '../src/notifier'
```

Comment:
Change to test against src rather than dist or coverage doesn't report correctly. As a consequence we lose the types, hence the colocated definitions file above that re-exports the types.